Add Datadog sample event data (#373)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/datadog.md

Done! any problem detected on your Datadog instance will trigger a webhook event. Port will parse the events according to the mapping and update the catalog entities accordingly.

> ## Let's Test It

Reviewer: ```suggestion
## Test the webhook
```

> In this section, we'll explore the webhook event data that is received from Datadog whenever an alert is triggered. We'll also delve into how the entity is finally created in Port by using the webhook configuration.

Reviewer: ```suggestion
This section includes a sample webhook event sent from Datadog when an alert is triggered. In addition, it also includes the entity created from the event based on the webhook configuration provided in the previous section.
```

### Payload

> Below is an example of the payload structure sent to the webhook URL after a monitor or alert is created:

Reviewer: ```suggestion
Here is an example of the payload structure sent to the webhook URL when a Datadog monitor or alert is created:
```

<details>
<summary> Webhook event payload</summary>

```json showLineNumbers
{
  "id": "7050278235293370890",
  "message": "Webhook message text",
  "priority": "normal",
  "last_updated": "1684492195000",
  "event_type": "query_alert_monitor",
  "event_url": "https://us5.datadoghq.com/event/event?id=7050278235293370890",
  "service": "api-service",
  "creator": "user@domain.com",
  "title": "[Triggered] [TEST] Service with IP 172.19.128.1 have exceeded memory limit",
  "date": "1684492195000",
  "org_id": "1300048894",
  "org_name": "Port",
  "alert_id": "147793",
  "alert_metric": "system.mem.used",
  "alert_status": "system.mem.used over host:api-service was > 7516192768.0 on average during the last 5m.",
  "alert_title": "[TEST] api-service with IP 172.19.128.1 have exceeded memory limit",
  "alert_type": "error",
  "tags": "host:api-service,monitor"
}
```

</details>

### Mapping Result

> Using the mappings defined in the webhook configuration, Port will extract the necessary properties from the Datadog webhook payload and use the resulting data to create the alert entities. Below is the result of the mapping:

Reviewer: ```suggestion
The combination of the sample payload and the webhook configuration generate the following Port entity:
```
